ui/app/serializers/application.js

```diff
@@ -15,7 +15,14 @@ export default DS.JSONSerializer.extend({
           return key;
         }
         let pk = this.get('primaryKey') || 'id';
-        return { [pk]: key };
+        let model = { [pk]: key };
+        // if we've added a in the adapter, we want
```

Comment:
missing a word here I think

Reply:
Oops, I’ll fix this up post-merge, looks like I flubbed an edit 😅